internal/kubernetes/client/client.go

```diff
@@ -85,6 +85,10 @@ func Init() error {
 			return
 		}
 		flag.Parse()
+
+		if initErr == nil {
+			utils.PrintLog("info", utils.LogLine{Message: "init", Category: "kubernetes", Status: utils.SuccessStr})
```

Comment:
same as in aws check, maybe better error?

Reply:
it's a success not an error, this is line in charge to print in the stdout that the init went well